cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -141,6 +145,10 @@ public synchronized void create(final NamespaceMeta metadata) throws Exception {
       authorizerInstantiator.get().enforce(instanceId, principal, Action.ADMIN);
     }
 
+    // Skip authorization grants for the system user
+    if (!(Principal.SYSTEM.equals(principal) && NamespaceId.DEFAULT.equals(namespace))) {
```

Comment:
Collapse this code snippet with above statement, which has a comment's explanation. The DEFAULT namespace is created by `DefaultNamespaceEnsurer` thread at startup and we skip the authorization process for that.